yt/visualization/plot_container.py

```diff
@@ -1,12 +1,13 @@
 import base64
 import builtins
 import os
+import textwrap
 from collections import defaultdict
 from functools import wraps
 
 import matplotlib
 import numpy as np
-from matplotlib.cm import get_cmap
+from matplotlib.cm import get_cmap as get_cmap
```

Comment:
Is this necessary?

Reply:
nope, it's clearly a leftover, good catch